benchmark/compare/main.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func parser(s string) []string {
+	start, end := 0, 0
+	leng := len(s)
+	var result []string
+	for end < leng {
+		for end < leng && (s[end] == ' ' || s[end] == '\t') {
+			end++
+		}
+		start = end
+		for end < leng && s[end] != ' ' && s[end] != '\t' {
+			end++
+		}
+		tmp := s[start:end]
+		start = end
+		result = append(result, tmp)
+	}
+	return result
+}
+
+func createMap(fileName string, m map[string][]string) {
+	f, err := os.Open(fileName)
+	if err != nil {
+		fmt.Println("read file err")
```

Comment:
Print the error content as well
`fmt.Printf("failed to read file: %v\n", err)`

And should probably just return here.